test/integration/enumerate_databases/enumerate_databases.prose.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
+
+import type { MongoClient } from '../../../src';
+
+describe('listDatabases() spec prose', function () {
+  /**
+   * Execute the method to enumerate full database information (e.g. listDatabases())
+   * - Verify that the method returns an Iterable of Document types
+   * - Verify that all databases on the server are present in the result set
+   * - Verify that the result set does not contain duplicates
+   */
+  let client: MongoClient;
+
+  beforeEach(async function () {
+    client = this.configuration.newClient();
+    await client.connect();
+  });
+
+  afterEach(async function () {
+    await client?.close();
+  });
+
+  it('Verify that the method returns an Iterable of Document types', async () => {
+    const dbInfo = await client.db().admin().listDatabases();
+    expect(dbInfo).to.have.property('databases');
+    expect(dbInfo.databases).to.be.an('array');
+    expect(dbInfo.databases).to.have.lengthOf.at.least(1);
+    for (const db of dbInfo.databases) {
+      expect(db).to.be.a('object');
+    }
+  });
+
+  it('Verify that all databases on the server are present in the result set', async () => {
```

Comment:
We should try to make sure that we have a clean environment with a known list of dbs for this test, and then assert on the expected list

Reply:
done!